### PR TITLE
Allow setting of custom tag values

### PR DESCRIPTION
**Pull request by anfleene**

### CODE REVIEW

File: .kitchen.yml [hunk 1]
```diff
@@ -37,6 +37,7 @@ provisioner:
             weekday: '6'
           command:
             tag: 'Backups'
+            tag_value: 'weekly'
```
Comment (bbergstrom) [inline .kitchen.yml]: Consider adding or extending the test suite to test for the backwards compatibility of existing tags.

Comment (anfleene) [inline .kitchen.yml]: Every other command uses the default tag with no value

Comment (bbergstrom) [inline .kitchen.yml]: k

**Comment by bbergstrom** [comment]
:+1:

**Pull request merged**